.azurePipeline/unittest_wheel_steps.yml

```diff
@@ -35,8 +35,9 @@ steps:
       opSysFlag=$(echo ${{ parameters.operatingSystem }} | sed 's/[[:punct:]]//g' | tr '[:upper:]' '[:lower:]' | head -c30)
       pyVFlag=$(echo python${{ parameters.pythonVersion }} | sed 's/[[:punct:]]//g' | tr '[:upper:]' '[:lower:]' | head -c30)
       echo "codecov flags: $opSysFlag,$pyVFlag"
-      python -m codecov --token $(CODECOV_TOKEN) \
-        --file coverage.xml \
+      bash <(curl -s https://codecov.io/bash) \
```

Comment:
no need for token anymore?

Reply:
the codecov tool reads the token from the environment variable. So no need to specify it here. -> Less clutter. :)